server/graphql/v2/input/TierCreateInput.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  GraphQLBoolean,
+  GraphQLInputObjectType,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLString,
+} from 'graphql';
+
+import { TierAmountType, TierType } from '../enum';
+import { TierFrequency } from '../enum/TierFrequency';
+
+import { AmountInput } from './AmountInput';
+
+export const TierCreateInput = new GraphQLInputObjectType({
+  name: 'TierCreateInput',
+  fields: () => ({
+    amount: {
+      type: new GraphQLNonNull(AmountInput),
+    },
+    name: {
+      type: new GraphQLNonNull(GraphQLString),
```

Comment:
:ice_cream: Not a requested change: I stumbled upon the `NonEmptyString` scalar (from `graphql-scalar`) this week. Could be interesting for this kind of field.